app/views/idv/phone/new.html.slim

```diff
@@ -12,7 +12,7 @@ ul.py1.m0
 = simple_form_for(@idv_form, url: idv_phone_path,
     html: { autocomplete: 'off', method: :put, role: 'form', class: 'mt2' }) do |f|
   = f.label :phone, label: t('idv.form.phone'), class: 'bold'
-  = f.input :phone, required: true, input_html: { class: 'us-phone' }, label: false,
+  = f.input :phone, required: true, input_html: { class: 'us-phone sm-col-8' }, label: false,
```

Comment:
Is this the correct place to put this class? I didn't understand where the original html was for this input form and maybe it should go there instead.